mappings/net/minecraft/block/AbstractFurnaceBlock.mapping

```diff
@@ -3,7 +3,7 @@ CLASS net/minecraft/class_2363 net/minecraft/block/AbstractFurnaceBlock
 	FIELD field_11105 LIT Lnet/minecraft/class_2746;
 	METHOD <init> (Lnet/minecraft/class_2248$class_2251;)V
 		ARG 1 settings
-	METHOD method_17025 openContainer (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;)V
+	METHOD method_17025 openScreen (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;)V
```

Comment:
Arguably I feel openScreen(ScreenHandler) sounds far to similar to MinecraftClient's openScreen(Screen)

Reply:
That's sorta the point though. Since opening a handler opens a screen on the client. It's literally just the serverside way of opening a screen.